src/apps/Console/ConsoleFeedback.js

```diff
@@ -21,325 +28,269 @@ export default function ConsoleFeedback({
           display: flex;
           align-items: center;
           justify-content: center;
-          padding: ${GU * 6}px 0 ${GU * 6}px 0;
+          padding: ${6 * GU}px 0 ${6 * GU}px 0;
         `}
       >
         <LoadingRing />
         <span
           css={`
             margin-left: ${1 * GU}px;
-            ${textStyle('body2')}
+            ${textStyle('body2')};
           `}
         >
-          Executing Command...
+          Executing command...
```

Comment:
Maybe we wanna use the Unicode equivalent of the horizontal ellipsis here: `…`

Reply:
Done in 9e9739b.